openadapt/record.py

```diff
@@ -167,6 +175,9 @@ def process_events(
     started = False
     while not terminate_processing.is_set() or not event_q.empty():
         event = event_q.get()
+        if event.type == "window" or event.type == "a11y":
+            handle = event.data["handle"]
+            counter = event.data["counter"]
```

Comment:
What do you think about naming these more verbosely, e.g. `window_handle` and `window_counter`? Also, what do you think about defining these outside of the loop, setting them to `None` for improved readability?